docs/user-guide/rules.md

```diff
@@ -388,196 +388,4 @@ Require or disallow whitespace on the inside with this `whitespace-inside` rule.
 | | | |
 | :-- | :-: | :-: |
 | [`comment-whitespace-inside`](../../lib/rules/comment-whitespace-inside/README.md)<br/>Require or disallow whitespace on the inside of comment markers. | ✅ | 🔧 |
-
-## Deprecated
```

Comment:
questions

Are the plugins storing the documentation of the rules? (stylistic and codeguide)
If not could we at least archive all the READMEs of the removed rules in a folder somewhere?

Reply:
Do you mean the following?

https://github.com/elirasza/stylelint-stylistic/blob/0.4.2/lib/rules/at-rule-name-case/README.md